fuse_core/test/test_transaction.cpp

```diff
@@ -339,14 +339,14 @@ TEST(Transaction, Empty)
     EXPECT_FALSE(transaction.empty());
   }
 
-  // A transaction with involved stamps cannot be empty
+  // A transaction with involved stamps but no added or removed variables or constraints is empty
```

Comment:
A transaction with involved stamps only doesn't make much sense though. Not sure if there's an easy and cheap way to enforce that. I believe it's better to leave it as is.